chai-as-promised/chai-as-promised.d.ts

```diff
@@ -32,4 +32,4 @@ declare module chai {
         isRejected(promise: any, expected: any, message?: string): void;
         isRejected(promise: any, match: RegExp, message?: string): void;
     }
-}
\ No newline at end of file
+}
```

Comment:
Hmm, my diff in github shows that it was removed... (no newline added). I like the change to Assertion, I think that clears up a lot. :+1: